INSTALL.adoc

```diff
@@ -15,6 +15,7 @@ https://github.com/jonas/tig/issues/337[#337]
 
 The quick and simple way to install Tig is:
 
+	$ ./autogen.sh
```

Comment:
The default install method does not require autoconf so there's no need to run autogen.